picsartstyle.css

```
@@ -0,0 +1,415 @@
+    font-size: 20px;
+    position: relative;
+    left: 220px;
+    top: -59px;
+    position: relative;
+    left: 190px;
+}
+.Menu2{
+    margin: 10px;
+    
+}
+.Maincontainer{
+    box-sizing: border-box;
+    display: flex;
+    flex: 0 1 auto;
+    flex-direction: row;
+    flex-wrap: wrap;
+    margin-right: -8px;
+    margin-left: -8px;
+    position: relative;
+    top: -50px;
+}
+.Maincontainer1 {
+    position: relative;
+    
+}
+.Maincontainer3{
+    font-family: 'Urbanist', sans-serif;
+    font-size: 50px;
+    line-height: 70px;
+    color: #1F1F33;
+    text-align: initial;
+    font-weight: 500;
+    max-width: 32.666667%;;
+    
+}
+.Maincontainer4{
+    font-family: 'Urbanist', sans-serif;
+    font-size: 24px;
+    text-align: initial;
+    line-height: inherit;
+    color: #1F1F33;
+    max-width: 33.666667%;
+    position: relative;
+    bottom: 50px
+}
+.Maincontainer4-button{
+    background: #1F1F33;
+    border-radius: 10px;
+    display: flex;
+    align-items: center;
+    text-align: center;
+    text-transform: capitalize;
+    color: #FCEECA;
+    margin-top: 5%;
+    cursor: pointer;
+    border-color: #6c757d;
+    border: 1px solid transparent;
+    font-size: 1rem;
+    padding: .375rem .75rem;
+}
+.Container-image{
+    border-radius: 50%;
+    box-shadow:  rgb(255 250 217) 110px 0px 0px 12px;
+    float: left;
+    position: relative;
+    left: 50%;
+    bottom: 458px;
+    padding-left: 15px;
+    
+}
+.Container-image .image{
+    border-radius: 50%;
+}
+@media only screen and (min-width: 992px){
+.Container {
+    width: 976px;
+    max-width: 100%;
+}
+}
+@media (min-width:768px) {
+ .Maincontainer2{
+    margin-left: 8.666667%;
+ }   
+}
+@media only screen and (min-width: 576px){
+    .Maincontainer{
+        flex-direction: column;
+    }
+}
+.Container-a{
+    background: #FFFDED;
+    height: 454px;
+    position: relative;
+    top: 145px;
+    margin: -8px;
+}
+.Container-font{
+    position: relative;
+    left: -450px;
+    bottom: 30px;
+}
+.Container-font h2{
+    font-family: Museo Sans;
+    font-size: 32px;
+    line-height: 44px;
+    padding-top: 10%;
+    text-align: center;
+    color: #333333;
+    font-weight: 500;
+}
+.Container-font p{
+    font-family: Museo Sans;
+    font-size: 20px;
+    line-height: 28px;
+    text-align: center;
+    padding-top: 2%;
+    color: #333333;
+    font-weight: 500;
+    position: relative;
+    bottom: 30px;
+}
+@media only screen and (min-width: 768px){
+    .Container-font  {
+        max-width: 65.666667%;
+        box-sizing: border-box;
+        flex: 0 0 auto;
+        padding-right: 8px;
+        padding-left: 8px;
+        margin-left: 46.666667%;
+        position: relative;
+        width: 100%;
+    }
+}
+.Button-Aboutus{
+    position: relative;
+    left: 430px;
+    margin-top: 3%;
+    cursor: pointer;
+    color: #6c757d;
+    border-color: #6c757d;
+    background-color: transparent;
+    border: solid 1px rgba(0, 0, 0, 0.5);
```

Fist should be the thickness and then border style
```suggestion
    border: 1px solid rgba(0, 0, 0, 0.5);
```